ios/browser/api/ads/brave_ads.h

```diff
@@ -134,9 +134,9 @@ OBJC_EXPORT
 - (void)reportTabClosedWithTabId:(NSInteger)tabId
     NS_SWIFT_NAME(reportTabClosed(tabId:));
 
-/// Report that an ad notification event type was triggered for a given id
-- (void)reportAdNotificationEvent:(NSString*)placementId
-                        eventType:(AdsAdNotificationEventType)eventType;
+/// Report that a notification ad event type was triggered for a given id
+- (void)reportNotificationAdEvent:(NSString*)placementId
+                        eventType:(AdsNotificationAdEventType)eventType;
```

Comment:
I don't see where this is renamed in iOS side?

Reply:
Fixed